airflow-core/src/airflow/api_fastapi/core_api/routes/ui/grid.py

```diff
@@ -161,15 +161,16 @@ def get_dag_structure(
         return nodes
 
     serdags = session.scalars(
-        select(SerializedDagModel).where(
+            select(SerializedDagModel).where(
+            and_(SerializedDagModel.dag_id == dag_id,
```

Comment:
Nit: `and_` is the default, no need to use it here:
```suggestion
        select(SerializedDagModel).where(
            SerializedDagModel.dag_id == dag_id,
```